Src/WitsmlExplorer.Api/Workers/DownloadAllLogDataWorker.cs

```diff
@@ -1,6 +1,7 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Text;
```

Comment:
System.Text is not used and shouldn't be added.

Reply:
fixed